command-expansion-server/test/__snapshots__/command-types.spec.ts.snap

```diff
@@ -299,20 +297,16 @@ export class Sequence {
   }
 
   public toEDSLString(): string {
+    const commandsString =
+      this.commands.length === 0 ? '' : '\\n' + indent(this.commands.map(c => c.toEDSLString() + ',').join('\\n'), 3);
 
-    const commandsString = this.commands.length === 0
-        ? ''
-        : '\\n' + indent(this.commands.map(c => c.toEDSLString() + ',').join('\\n'), 3);
-
-  return (
-\`export default () =>
+    return \`export default () =>
   Sequence.new({
     seqId: '\${this.seqId}',
     metadata: \${JSON.stringify(this.metadata)},
     commands: [\${commandsString}
     ],
-  });\`
-  );
```

Comment:
Why was this return wrapped in `();`?

Reply:
So this is the generated file but the [file that generates this code](https://github.com/NASA-AMMOS/aerie/pull/409/files#diff-224d12ae69810ebad6855e95127d0a73a31d2653d5207744b5f8f68370bc0727L304) had parens in the `return` statement. They are not needed tho so Prettier removed them.